usaspending_api/api_contracts/contracts/v2/download/awards.md

```diff
@@ -164,3 +164,7 @@ This route sends a request to the backend to begin generating a zipfile of award
 + `city` (optional, string)
 + `district` (optional, string)
 + `zip` (optional, string)
+
+### NAICSCodeObject (object)
++ `require`: [`33`]
```

Comment:
These fields need the types to be defined instead of showing an example (`optional, array[string]`)

Reply:
This looks resolved, but the [`33`] should be in quotes as a string, yes?

Reply:
I don't think so. MSON isn't a very strict language https://apiblueprint.org/documentation/mson/specification.html#232-value-member-types